llvm/test/tools/llvm-objdump/Offloading/fatbin-offloading.test

```diff
@@ -0,0 +1,80 @@
+// RUN: clang++ -x hip --hip-link --offload-arch=gfx1100 --offload-arch=gfx1101 -o %t %s
```

Comment:
Due to layering (LLVM_ENABLE_PROJECTS may not include clang), llvm/test tests cannot use clang

Reply:
I'm actually going to just remove this test, in favour of using the other one, which just uses yaml2obj.